modules/core/client/components/README.md

```diff
@@ -1,5 +1,36 @@
 # Reusable components
 
+## Navigation
```

Comment:
Good point! What was fine in `references` module suddenly becomes way too generic in `core`. I'll go with `StepNavigation`, because when I hear `Wizard`, I imagine Gandalf or Harry Potter.

Reply:
<h1>🧙‍♂️</h1>

Reply:
@simison, I haven't realized `Wizard` is a common term. I've done the renaming to `StepNavigation` in a25fa4fc396a02ee0d61e3c918a748331df6affd, but if you strongly prefer `WizardNavigation`, let me know. It doesn't matter to me.

Reply:
`Step*` sounds great and clear 👌